packages/core/src/state-management/repository.ts

```diff
@@ -265,6 +284,36 @@ export class Repository {
     return state$
   }
 
+  async resumeRunningStatesFromAnchorRequestStore(): Promise<void> {
```

Comment:
Should it really be a part of repository?

Reply:
IMO it would better be a separate service that uses the repository.

Reply:
Good call, will do!